### PR TITLE
adding welsh census branding

### PR DESCRIPTION
**Pull request by yomiodeleye**
### What
Adding Welsh census branding on new release page template

### How to review

When you land on the release page and you change local to welsh, the census logo should change with Welsh census logo

### Who can review

FD

### CODE REVIEW

File: assets/templates/partials/release/contents/about-the-data.tmpl [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
+            width="171"
+          >
+        {{ else }}
+          <img
+            src="https://cdn.ons.gov.uk/assets/images/census-logo/logo-cyfrifiad-2021-purple-landscape.svg"
+            alt="Census 2021 logo"
+            class="ons-u-mb-xs"
+            width="171"
+          >
+        {{ end }}
```
Comment (gilesburdett) [inline assets/templates/partials/release/contents/about-the-data.tmpl:18]: `if/else/end` indentation should be consistent.

File: assets/templates/partials/release/contents/about-the-data.tmpl [hunk 2]
```diff
@@ -1,11 +1,21 @@
 <div class="about-the-data">
   {{ if .Description.Census2021 }}
     <div class="ons-u-mb-l census-2021">
-      <img
-        src="https://cdn.ons.gov.uk/assets/images/census-logo/logo-census-2021-purple-landscape.svg"
-        alt="Census 2021 logo"
-        class="ons-u-mb-xs"
-      >
+      {{ if eq .Language "en" }}
```
Comment (gilesburdett) [inline assets/templates/partials/release/contents/about-the-data.tmpl:4]: The default language is English so this should be the fallback option in the `else` block, and this guard should test for `cy`.

Comment (gilesburdett) [reply assets/templates/partials/release/contents/about-the-data.tmpl:4]: And remove the trailing spaces

File: assets/templates/partials/release/status-header.tmpl [hunk 3]
```diff
@@ -34,10 +34,19 @@
           class="ons-u-pb-m"
         {{ end }}
       >
-        <img
-          src="https://cdn.ons.gov.uk/assets/images/census-logo/logo-census-2021-purple-landscape.svg"
-          alt="Census 2021 logo"
-        >
+        {{ if eq .Language "en" }}
```
Comment (gilesburdett) [inline assets/templates/partials/release/status-header.tmpl:37]: As above on
- trailing spaces 
- switching around so English is the fallback, not the guard

**Review by gilesburdett — CHANGES_REQUESTED** [review]

File: assets/templates/partials/release/contents/about-the-data.tmpl (hunk 1 above)
Comment (yomiodeleye) [reply assets/templates/partials/release/contents/about-the-data.tmpl:18]: This done.

File: assets/templates/partials/release/contents/about-the-data.tmpl (hunk 2 above)
Comment (yomiodeleye) [reply assets/templates/partials/release/contents/about-the-data.tmpl:4]: This is done now.

File: assets/templates/partials/release/status-header.tmpl (hunk 3 above)
Comment (yomiodeleye) [reply assets/templates/partials/release/status-header.tmpl:37]: this is done now.

**Review by gilesburdett — APPROVED** [review]

**Pull request merged**